{"name": "test_addenterprise03", "status": "failed", "statusDetails": {"message": "AssertionError: assert '成功' == '找不到指定记录'\n  - 找不到指定记录\n  + 成功", "trace": "self = <testcase.test_enteripse.Testenterprise object at 0x0453E1F0>\n\n    def test_addenterprise03(self):          #删除子企业员工\n        case_data = get_test_data(self.excel_data,'delenterprise')\n        updatetoken(self.token, case_data)\n        takeid = re.findall(r'http://api.etmprot.etmcn.com/basic/v2/employee/(.*)', case_data[\"url\"])\n        sumid = str(int(takeid[0]) + 1)\n        data = json.loads(case_data['data'])\n        id = {\"id\": sumid}\n        data.update(id)\n        case_data['data'] = json.dumps(data)\n        new_url = \"http://api.etmprot.etmcn.com/basic/v2/employee/\"+sumid\n        case_data[\"url\"] = new_url\n        res = send_request(case_data)\n>       assert case_data['expect_res'] == res[\"message\"]\nE       AssertionError: assert '成功' == '找不到指定记录'\nE         - 找不到指定记录\nE         + 成功\n\ntestcase\\test_enteripse.py:55: AssertionError"}, "start": 1600238709414, "stop": 1600238709628, "uuid": "e52c5be5-7080-4f52-9646-ed52ea6a97de", "historyId": "1faf892aa02ca937d2b5b0556fa49915", "testCaseId": "9a530e2c0efe1d1bbcc418f0f6e26dfc", "fullName": "testcase.test_enteripse.Testenterprise#test_addenterprise03", "labels": [{"name": "parentSuite", "value": "testcase"}, {"name": "suite", "value": "test_enteripse"}, {"name": "subSuite", "value": "Testenterprise"}, {"name": "host", "value": "TX5C3S1L7D31UYD"}, {"name": "thread", "value": "8104-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testcase.test_enteripse"}]}